client/blocks/upgrade-nudge-expanded/docs/example.jsx

```diff
@@ -0,0 +1,40 @@
+/** @format */
```

Comment:
The format pragma is no longer needed and can be removed (see also https://github.com/Automattic/wp-calypso/pull/36353).